bundles/org.eclipse.e4.ui.workbench3/META-INF/MANIFEST.MF

```diff
@@ -10,6 +10,7 @@ Require-Bundle: org.eclipse.equinox.common;bundle-version="[3.7.0,4.0.0)",
  org.eclipse.swt;bundle-version="[3.6.0,4.0.0)",
  org.eclipse.equinox.registry;bundle-version="[3.5.0,4.0.0)",
  org.eclipse.core.runtime;bundle-version="[3.29.0,4.0.0)"
-Export-Package: org.eclipse.ui.testing,
+Export-Package: org.eclipse.ui.internal.activitysupport,
```

Comment:
Is there any reason why the API is inside **internal** package: org.eclipse.ui.**internal**.activitysupport ? Also I'm not quite sure about the target bundle `org.eclipse.e4.ui.workbench3`, that only contains some very special `org.eclipse.ui.testing` code.

What about `org.eclipse.e4.ui.activitysupport` package inside `org.eclipse.e4.ui.workbench` bundle?

Reply:
> Is there any reason why the API is inside **internal** package: org.eclipse.ui.**internal**.activitysupport ? Also I'm not quite sure about the target bundle `org.eclipse.e4.ui.workbench3`, that only contains some very special `org.eclipse.ui.testing` code.
> 
> What about `org.eclipse.e4.ui.activitysupport` package inside `org.eclipse.e4.ui.workbench` bundle?

I wanted a bundle which is common to both `org.eclipse.e4.ui.workbench.renderers.swt` and `org.eclipse.ui.workbench`. Which does not create any cycle problem. I found  `org.eclipse.e4.ui.workbench3` worked.
Now I tried `org.eclipse.e4.ui.workbench` which also works fine. I will move new API here.